### PR TITLE
Base 2018

### PR DESCRIPTION
**Pull request by bobdenotter**
Let's go to beta! 

Screenshots: 

screenshot-2018-3-13 quodsi ipsam honestatem undique pertectam atque absolutam a sample site

-----------------

screenshot-2018-3-13 cur id non ita fit a sample site

-----------------

screenshot-2018-3-13 quam nemo umquam voluptatem appellavit appellat a sample site

-----------------

screenshot-2018-3-13 omnes enim iucundum motum quo sensus hilaretur a sample site

-----------------

screenshot-2018-3-13 a sample site the amazing payoff goes here 1

-----------------

screenshot-2018-3-13 a sample site the amazing payoff goes here

### CODE REVIEW

File: base-2018/partials/_footer.twig [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                    template = 'partials/_sub_menu_footer.twig',
+                    params = {'withsubmenus': false}
+                ) }}
+            </div>
+        </nav>
+        {{ widgets('footer') }}
+    </div>
+</footer>
+
+<script src="{{ asset('js/jquery.js', 'theme') }}"></script>
```
Comment (xiaohutai) [inline base-2018/partials/_footer.twig]: [Performance/Optimization] Concat and minify these four lines?

Comment (bobdenotter) [reply base-2018/partials/_footer.twig]: Maybe. I kinda like that people can _remove_ stuff if they don't like it. That said: magnific and brightness _do_ need to go into app.js, and I _might_ want to add Prism, for out-of-the-box pretty code snippets.

Comment (GawainLynch) [reply base-2018/partials/_footer.twig]: We could do something sensible, like realise that jQuery shouldn't be used post-2015, no?

File: base-2018/source/gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+// Browsers to target when prefixing CSS.
+var COMPATIBILITY = ['last 2 versions', 'ie >= 9'];
+
+// Define base paths for Sass and Javascript.
+// File paths to various assets are defined here.
+var PATHS = {
+  //assets: [
+  //  'src/assets/**/*',
+  //  '!src/assets/{img,js,scss}/**/*'
+  //],
```
Comment (GawainLynch) [inline base-2018/source/gulpfile.js]: Remove please

File: base-2018/index.twig [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                            <p>{{ page.excerpt }}</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            {% endfor %}
+        </div>
+        {# <div class="buttons has-addons is-right">
+            <a class="button is-primary" href="{{ path('contentlisting', {'contenttypeslug': 'pages'}) }}">Read more<span class="is-hidden"> Pages</span></a>
+        </div> #}
```
Comment (GawainLynch) [inline base-2018/index.twig]: Remove please

**Comment by zomars** [comment]
+1 for Bulima <3

**Comment by GawainLynch** [comment]
> Removed Jquery and Magnific. Added Prism and Baguettebox.

How awesome was it to wake to this today! :heart:

**Pull request merged**